portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/externalvideoshortcut/DMVideoShortcut.testcase

```diff
@@ -811,6 +811,12 @@ definition {
 			videoShortcutTitle = "Test Twitch Video",
 			videoURL = "https://www.twitch.tv/videos/806178790");
 
+		DMNavigator.openDocumentsAndMediaAdmin(siteURLKey = "guest");
+
+		Search.searchCP(searchTerm = "tag name");
+
+		DMDocument.viewAllDocumentsChecked(detailsMessage = "3 items are selected.");
+
```

Comment:
I don't think running an extra search query is necessary. Can we just add DMDocument.viewAllDocumentsChecked after Search.searchCP(searchTerm = "tag") in line 825?

Reply:
It's wrote like that because as we can see in [LPS-68055](https://issues.liferay.com/browse/LPS-68055) queries are send from documents and media admin. If put the code on 825 it would query from a Site page, so it may not cover the ticket. Let me know if I have to change it anyway!

Reply:
Ah you're right. Thanks @jaimelr10 !